### PR TITLE
chore: layer shell programs need privileged

### PR DESCRIPTION
**Pull request by cwegener**
see 6efb6b8

### CODE REVIEW

**Comment by cwegener** [comment]
I have no clue why this failed https://builds.sr.ht/~jorth/job/1200495#task-test-112

All tests pass when run locally.

File: docs/config.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ alt-shift-f = "toggle-floating"
 # For example, the exec action spawns an application and has the exec field
 # that describes how to spawn the application.
 Super_L = { type = "exec", exec = "alacritty" }
-alt-p = { type = "exec", exec = "bemenu-run" }
+alt-p = { type = "exec", exec = { prog: "bemenu-run", privileged = true } }
```
Comment (mahkoh) [inline docs/config.md]:
```suggestion
alt-p = { type = "exec", exec = { prog = "bemenu-run", privileged = true } }
```

**Comment by mahkoh** [comment]
>I have no clue why this failed [builds.sr.ht/~jorth/job/1200495#task-test-112](https://builds.sr.ht/~jorth/job/1200495#task-test-112)

Unfortunately the integration tests are flaky in CI. The artifacts are no longer available so I cannot check what failed this time.

**Review by mahkoh — CHANGES_REQUESTED** [review]

**Comment by mahkoh** [comment]
Thanks!

**Pull request merged**